sdk/src/org.graalvm.polyglot/src/org/graalvm/polyglot/impl/AbstractPolyglotImpl.java

```diff
@@ -803,7 +803,9 @@ public abstract Object createContext(Object receiver, SandboxPolicy sandboxPolic
                         Object hostAccess,
                         Object polyglotAccess,
                         boolean allowNativeAccess,
-                        boolean allowCreateThread, boolean allowHostClassLoading, boolean allowInnerContextOptions, boolean allowExperimentalOptions,
+                        boolean allowCreateThread,
+                        Consumer<String> onDeniedThreadAccess,
```

Comment:
No longer needed no?

Reply:
Removed in 29a5199. However, as I [argued here](https://github.com/oracle/graal/pull/8266#discussion_r1767009663) I am convinced there will be a time when the `Consumer<String>` will get into `org.graalvm.polyglot` as well. Then `git show 29a5199 | patch -R -p1` will be needed. But of course, the code doesn't need to be changed by this PR.

Reply:
I don't like to keep dead code. Thanks for removing.